35.md

```diff
@@ -0,0 +1,70 @@
+NIP-35
+======
+
+Torrents
+-----------
+
+`draft` `optional`
+
+This NIP defined a new `kind 2003` which is a Torrent. 
+
+`kind 2003` is a simple torrent index where there is enough information to search for content and construct the magnet link. No torrent files exist on nostr.
+
+## Tags
+- `x`: V1 BitTorrent Info Hash, as seen in the [magnet link](https://www.bittorrent.org/beps/bep_0053.html) `magnet:?xt=urn:btih:HASH`
+- `file`: A file entry inside the torrent, including the full path ie. `info/example.txt`
+- `tracker`: (Optional) A tracker to use for this torrent
+
+In order to make torrents searchable by general category, you SHOULD include a few tags like `movie`, `tv`, `HD`, `UHD` etc.
+
+## Tag prefixes
+
+Tag prefixes are used to label the content with references, ie. `["i", "imdb:1234"]`
+
+- `tcat`: A comma separated text category path, ie. `["i", "tcat:video,movie,4k"]`, this should also match the `newznab` category in a best effort approach.
+- `newznab`: The category ID from [newznab](https://github.com/Prowlarr/Prowlarr/blob/develop/src/NzbDrone.Core/Indexers/NewznabStandardCategory.cs)
+- `tmdb`: [The movie database](https://www.themoviedb.org/) id.
+- `ttvdb`: [TV database](https://thetvdb.com/) id.
+- `imdb`: [IMDB](https://www.imdb.com/) id.
+- `mal`: [MyAnimeList](https://myanimelist.net/) id.
+- `anilist`: [AniList](https://anilist.co/) id.
+
+A second level prefix should be included where the database supports multiple media types.
+- `tmdb:movie:693134` maps to `themoviedb.org/movie/693134`
+- `ttvdb:movie:290272` maps to `thetvdb.com/movies/dune-part-two`
+- `mal:anime:9253` maps to `myanimelist.net/anime/9253`
+- `mal:manga:17517` maps to `myanimelist.net/manga/17517`
+
+In some cases the url mapping isnt direct, mapping the url in general is out of scope for this NIP, the section above is only a guide so that implementers have enough information to succsesfully map the url if they wish.
+
+```jsonc
+{
+  "kind": 2003,
+  "content": "<long-description-pre-formatted>",
+  "tags": [
+    ["title", "<torrent-title>"],
+    ["x", "<bittorrent-info-hash>"],
+    ["file", "<file-name>", "<file-size-in-bytes>"],
+    ["file", "<file-name>", "<file-size-in-bytes>"],
+    ["tracker", "udp://mytacker.com:1337"],
+    ["tracker", "http://1337-tracker.net/announce"],
+    ["i", "tcat:video,movie,4k"],
+    ["i", "newznab:2045"],
+    ["i", "imdb:tt15239678"],
+    ["i", "tmdb:movie:693134"],
+    ["i", "ttvdb:movie:290272"],
+    ["t", "movie"],
+    ["t", "4k"],
+  ]
+}
+```
+
+## Torrent Comments
+
+A torrent comment is a `kind 2004` event which is used to reply to a torrent event.
+
+This event works exactly like a `kind 1` and should follow `NIP-10` for tagging.
```

Comment:
Save the ~planet~nostr, guys! Support #1233 with a :+1:? =D

Reply:
I have not read through the entirety of #1233 but I think there is merit in having a specific kind for torrent commends and using a simple schema like NIP-10 for marking replies and threads

It seems reasonable that a NIP would define its primary event kind (in this case `2003`) and a comment / reply kind that is specifically used for it. It seems like a better solution then using `k` tags as a "meta" kind

Reply:
Thread roots should be defined by the NIP. Replies can be generalized. 

Otherwise, should we have reactions, zap, labels, and report kinds for each NIP as well? Replies are no different than reactions.

Reply:
Seems reasonable, but im not sold on the new tag structure

Reply:
Going to merge anyway, we can always change it later once your PR is merged, nobody is really using comments on torrents yet. I only see 24 comments so far.